src/mixins/_embed.scss

```diff
@@ -84,20 +84,28 @@ $default-aspect-ratio: aspect-ratio.$square;
   }
 }
 
-@mixin object {
+/**
+ * We declare a "default value" for our mixin's `$ratio` argument so
+ * it may be used without passing any arguments. This allows us
+ * to override the "default" aspect ratio for use cases such as WordPress
+ * Gutenberg blocks, where it may be more desirable to re-define the default
+ * aspect-ratio instead of adding additional modifier classes.
+ */
+
+@mixin object($ratio: #{$default-aspect-ratio}) {
```

Comment:
I didn't catch this originally, but where does the mixin name `object` come from?

Reply:
I'm open to other naming options that improve clarity. 

I had originally arrived at `object` because when looking for an appropriate name, I considered how positioning and sizing of embedded content is controlled by `object-position` and `object-fit`. I _thought_ `object` was common, generic enough to describe a variety of embed-able content and made sense when read as `@include @embed.object`

from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/embed : 
**emphasis mine**

> You can use the **object**-position property to adjust the positioning of the embedded **object** within the element's frame, and the **object**-fit property to control how the **object's** size is adjusted to fit within the frame.

Other names I recall considering: 

- default
- element
- media

Reply:
When I've seen the term "object" used for a flexible embed component, it's been applied to the media element itself. In this case, it appears the mixin is applying styles to the flexible embed container, _not_ the media object/element/image/iframe/whatever.

I'm a fan of naming components what they _are_, and mixins what they _do_. Food for thought.

Reply:
> I'm a fan of naming components what they are, and mixins what they do. Food for thought.

That's logical - I like it.